src/runtime_src/xrt/device/hal.h

```diff
@@ -603,6 +603,12 @@ class device
     return operations_result<size_t>();
   }
 
+  virtual operations_result<size_t>
+  getSysfsPath(std::string subdev, std::string entry, char* path, int size)
```

Comment:
Fix per device.h comments

Reply:
changed to use `hal::operations_result<std::string>` in the new push